src/types/props.types.ts

```diff
@@ -0,0 +1,244 @@
+    foregroundColor: string | null;
+    color: string;
+    onChange: (param: TBarrierChangeParam) => void;
+    relative: boolean;
+    draggable: boolean;
+    lineStyle: string;
+    hidePriceLines: boolean;
+    high?: number;
+    low?: number;
+    hideBarrierLine?: boolean;
+    hideOffscreenBarrier?: boolean;
+    hideOffscreenLine?: boolean;
+    title?: string;
+    showOffscreenArrows?: boolean;
+    isSingleBarrier?: boolean;
+    opacityOnOverlap?: number;
+};
+
+export type TChartProps = {
+    requestAPI: BinaryAPI['requestAPI'];
+    requestSubscribe: BinaryAPI['requestSubscribe'];
+    requestForget: BinaryAPI['requestForget'];
+    requestForgetStream?: BinaryAPI['requestForgetStream'];
+    id?: string;
+    getMarketsOrder?: (active_symbols: ActiveSymbols) => string[];
+    getIndicatorHeightRatio?: TGetIndicatorHeightRatio;
+    symbol?: string;
+    feedCall?: { activeSymbols?: boolean; tradingTimes?: boolean };
+    granularity?: TGranularity;
+    chartType?: string;
+    startEpoch?: number;
+    endEpoch?: number;
+    chartControlsWidgets?: TChartControlsWidgets;
+    topWidgets?: () => React.ReactElement;
+    bottomWidgets?: () => React.ReactElement;
+    toolbarWidget?: () => React.ReactElement;
+    isMobile?: boolean;
+    onSettingsChange?: (newSettings: Omit<TSettings, 'activeLanguages'>) => void;
+    stateChangeListener?: TStateChangeListener;
+    settings?: TSettings;
+    barriers?: TBarrierUpdateProps[];
+    enableRouting?: boolean;
+    enable?: boolean;
+    isConnectionOpened?: boolean;
+    onMessage: (message: TNotification) => void;
+    isAnimationEnabled?: boolean;
+    showLastDigitStats?: boolean;
+    scrollToEpoch?: number | null;
+    scrollToEpochOffset?: number;
+    clearChart?: () => void;
+    onExportLayout?: (currentLayout: typeof CIQ.UI.Layout) => void;
+    importedLayout?: typeof CIQ.UI.Layout;
+    shouldFetchTradingTimes?: boolean;
+    maxTick?: number | null;
+    crosshair?: number | null;
+    crosshairTooltipLeftAllow?: number | null;
+    zoom?: number;
+    yAxisMargin?: { bottom: number; top: number };
+    enableScroll?: boolean | null;
+    enableZoom?: boolean | null;
+    initialData?: TInitialChartData;
+    chartData?: TInitialChartData;
+    networkStatus?: TNetworkConfig;
+    refreshActiveSymbols?: ChartState['refreshActiveSymbols'];
+    chartStatusListener?: ChartState['chartStatusListener'];
+    enabledChartFooter?: boolean;
+    anchorChartToLeft?: boolean;
+    margin?: number;
+    isStaticChart?: ChartState['isStaticChart'];
+    enabledNavigationWidget?: boolean;
+    onCrosshairChange?: (state?: number | null) => void | null;
+    crosshairState?: number | null;
+    children?: React.ReactNode;
+};
+
+export type TQuote = {
+    Date: string;
+    Open?: number;
+    High?: number;
+    Low?: number;
+    Close: number;
+    tick?: TicksStreamResponse['tick'];
+    ohlc?: OHLCStreamResponse['ohlc'];
+    DT?: Date;
+    prevClose?: number;
```

Comment:
```suggestion
    prevClose?: number;
    Volume?: number;
```

*I suggest adding 'Volume' based on ChartIQ's OHLC object format: https://documentation.chartiq.com/tutorial-InputDataFormat.html